qemu/tests/blockdev_commit_fio.py

```diff
@@ -47,6 +51,8 @@ def run_test(self):
             self.verify_data_file()
             bg_test.join()
```

Comment:
No need to join(), is it ?
As our discussion offline, `time_base` can guarantee fio is running when commit snapshot start (time_base > sleep_max),  it is not must to make fio running until `time_base` reached.